README.md

```diff
@@ -114,6 +114,9 @@ We also have explicit _non-goals_ for Carbon, notably including:
 
 -   A stable ABI for the entire language and library
 -   Perfect backwards or forwards compatibility
+-   Support legacy code for which source code is unavailable
+-   Follow C++'s compilation and linking model
```

Comment:
It isn't clear what this means by "compilation and linking model".

If it means "textual inclusion of header files", then definitely a non-goal.

If it means "fits into the compile and link facilities provided by widely used C++ build systems", then I think it is actually a *goal*, and a really important one.

And it might mean something in-between like the object file formats or binary module file formats, where I think its not clear whether its a goal or non-goal...

So I'm worried about including this as I don't really know how to make the wording sufficiently precise to get at which layer. And it isn't clear that the first thing (textual inclusion) is critical to call out here.